examples/net_vm/terraform.tfvars

```diff
@@ -1,8 +1,8 @@
 #access_key_id       = "MyAccessKey"
 #secret_key_id       = "MySecretKey"
 #region              = "eu-west-2"
+#image_id        = "ami-68ed4301" # Debian-11-2022.03.10-0 on eu-west-2
 
```

Comment:
is it gonna work without image_id ?

Reply:
Yes, the image_id has been exported to the yml file ``` TF_VAR_image_id: ${{ secrets.OSC_OMI_EXEMPLE }} ```